### PR TITLE
Christian's Review of v -21

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR addresses Christian's review of version -21, archived at: https://mailarchive.ietf.org/arch/msg/core/tPPSsrFMPZId_7Q7OrmbgN1DCdw/

### CODE REVIEW

**Comment by chrysn** [comment]
Thanks, LGTM where I may say so.

File: draft-ietf-core-oscore-groupcomm.md [hunk 1]
```diff
@@ -1387,7 +1388,7 @@ The pairwise mode does not support external verifiers of source authentication a
 
 An endpoint implementing only a silent server does not support the pairwise mode.
 
-Endpoints using the CoAP Echo Option {{RFC9175}} and/or block-wise transfers {{RFC7959}} in a group where the AEAD Algorithm and Pairwise Key Agreement Algorithm are set MUST support the pairwise mode. This applies, for example, to block-wise exchanges after a first block-wise request which targets all servers in the group and includes the CoAP Block2 option (see Section 3.8 of {{I-D.ietf-core-groupcomm-bis}}). This prevents the attack described in {{ssec-unicast-requests}}, which leverages requests sent over unicast to a single group member and protected in group mode.
+Endpoints using the CoAP Echo Option {{RFC9175}} in a group where the AEAD Algorithm and Pairwise Key Agreement Algorithm are set MUST support the pairwise mode. This prevents the attack described in Section 13.9, which leverages requests sent over unicast to a single group member and protected in group mode.
```
Comment (rikard-sics) [inline draft-ietf-core-oscore-groupcomm.md]: It's better to have the reference to section 13.9 here use markdown references instead. It should be referring to section _{#ssec-unicast-requests}_.

**Review by rikard-sics — APPROVED** [review]
Except for my comment below things look good to me, as aligned with our internal discussions and the discussion we had with Christian.

File: draft-ietf-core-oscore-groupcomm.md (hunk 1 above)
Comment (marco-tiloca-sics) [reply draft-ietf-core-oscore-groupcomm.md]: Thanks, now fixed in https://github.com/core-wg/oscore-groupcomm/pull/108/commits/d95ec712b8425e6e39601017db82ccfb1c0146ef

**Pull request merged**